### ISSUE TITLE
docker container got killed at night

### ISSUE THREAD

**Issue by dabenzel** [MrBrax/TwitchAutomator#137]
The docker-container had an exception at nearly 1h in the night

logs:

```
2022-04-27 01:00:17.466 | vodclass <INFO> VOD JSON xxxxx_2022-04-26T16_53_53Z_46275126637 exists (again?) change
No clients connected to broker for data: {"server":true,"action":"vod_updated","data":{"vod":{"basename":...
/usr/local/share/twitchautomator/server/src/Core/Log.ts:79
throw new Error("Log folder does not exist!");

^

Error: Log folder does not exist!

at Function.logAdvanced (/usr/local/share/twitchautomator/server/src/Core/Log.ts:79:19)
at Automator.download (/usr/local/share/twitchautomator/server/src/Core/Automator.ts:717:13)
at Automator.handle (/usr/local/share/twitchautomator/server/src/Core/Automator.ts:189:13)

Node.js v17.9.0
error Command failed with exit code 1.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by MrBrax** [comment]
are you using --dataroot:\\host\twitchvods like a directly mounted network share?

**Comment by dabenzel** [comment]
i use unraid and mounted it like this:
![image](https://user-images.githubusercontent.com/2480235/165563517-62828a01-a9da-41dc-8b20-4c0901c2cec3.png)


this is inside the container:

```
# ls -lisa /usr/local/share/twitchautomator/data
total 0
20481662 0 drwxrwxrwx 2   99 users   0 Apr 27 15:22 .
   29724 0 drwxrwxr-x 1 node node  642 Apr 26 18:25 ..
20481666 0 drwxrwxrwx 2   99 users   0 Apr 24 17:39 cache
20481665 0 drwxrwxrwx 2   99 users   0 Apr 24 18:43 config
20481673 0 drwxrwxrwx 2   99 users   0 Apr 27 00:00 logs
20481675 0 drwxrwxrwx 2   99 users   0 Apr 23 11:54 payloads
20481676 0 drwxrwxrwx 2   99 users   0 Apr 24 12:55 storage
# 
```

**Comment by dabenzel** [comment]
Have tried it and will have a look

**Comment by KurtzPT** [comment]
I was about to report the same issue.

**Comment by MrBrax** [comment]
> I was about to report the same issue, but different error:
> 
> ```
> No clients connected to broker for data: {"server":true,"action":"end_download","data":{"vod":{"basename"...
> /usr/local/share/twitchautomator/server/src/Core/TwitchVOD.ts:2510
> reject(new Error("twitchdownloadercli not found"));
> 
> ^
> 
> Error: twitchdownloadercli not found
> 
> at <anonymous> (/usr/local/share/twitchautomator/server/src/Core/TwitchVOD.ts:2510:24)
> at new Promise (<anonymous>)
> at Function.downloadChatTD (/usr/local/share/twitchautomator/server/src/Core/TwitchVOD.ts:2505:16)
> at _TwitchVOD.downloadChat (/usr/local/share/twitchautomator/server/src/Core/TwitchVOD.ts:2122:26)
> at Automator.download (/usr/local/share/twitchautomator/server/src/Core/Automator.ts:729:22)
> at runMicrotasks (<anonymous>)
> at processTicksAndRejections (node:internal/process/task_queues:96:5)
> at Automator.handle (/usr/local/share/twitchautomator/server/src/Core/Automator.ts:189:13)
> 
> Node.js v17.9.0
> error Command failed with exit code 1.
> 
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```
> 
> This was before you fixed #130, not sure if that commit fixed it.

it should be fixed, i try/catch that download now
bit weird that the binary isn't found though
